target/linux/ath79/dts/qca9563_rosinson_wr818.dts

```diff
@@ -123,29 +123,15 @@
 };
 
 &usb0 {
-	#address-cells = <1>;
-	#size-cells = <0>;
 	status = "okay";
-
-	port@1 {
-		reg = <1>;
-		#trigger-source-cells = <0>;
-	};
 };
 
 &usb_phy1 {
 	status = "okay";
 };
 
 &usb1 {
-	#address-cells = <1>;
-	#size-cells = <0>;
 	status = "okay";
-
-	port@2 {
-		reg = <2>;
```

Comment:
This is replaced with `reg = <1>` from the SoC dtsi file. Is that fine?

Reply:
I think it's okay. This device does not have any usb led or any usb hub chip inside. The port number should be 1, just like other qca9563 devices.
https://www.right.com.cn/forum/thread-372995-1-1.html

Some other similar change details:
ar9331_arduino_yun.dts: does not have hub chip
ar9344_wd_mynet-n750.dts: does not have usb led and hub chip
ar9344_winchannel_wb2000.dts: does not have hub chip and only 1 usb here